packages/core/src/domain/session/sessionState.ts

```diff
@@ -1,22 +1,40 @@
-import { isEmptyObject } from '../../tools/utils/objectUtils'
 import { objectEntries } from '../../tools/utils/polyfills'
 import { dateNow } from '../../tools/utils/timeUtils'
 import { SESSION_EXPIRATION_DELAY } from './sessionConstants'
 
 const SESSION_ENTRY_REGEXP = /^([a-z]+)=([a-z0-9-]+)$/
 const SESSION_ENTRY_SEPARATOR = '&'
 
+export const enum SessionExpiredReason {
+  UNKNOWN = '0',
+}
+
 export interface SessionState {
   id?: string
   created?: string
   expire?: string
   lock?: string
+  expired?: SessionExpiredReason
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I have few suggestions:
- I found confusing that an expired session is `{expired = '0'}` I would expect `expired=true` or `expired=1`. 
- The name "expired" is quite close to "expire" maybe we could name it "isExpired" to better distinguish them
- Maybe we could avoid having the expire reason since for now we only have UNKNOWN. If we want to anticipate keeping the reason we could use `ìsExpired=1` and introduce the enum with 2, and 3 later